### PR TITLE
Feature/jansson/version v2.13.1

### PR DESCRIPTION
**Pull request by ParianGol**

### CODE REVIEW

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 1]
```diff
@@ -0,0 +1,5965 @@
...
+
+You should also get your employer (if you work as a programmer) or school, if any, to sign a "copyright disclaimer" for the program, if necessary. For more information on this, and how to apply and follow the GNU GPL, see <https://www.gnu.org/licenses/>.
+
+The GNU General Public License does not permit incorporating your program into proprietary programs. If your program is a subroutine library, you may consider it more useful to permit linking proprietary applications with the library. If this is what you want to do, use the GNU Lesser General Public License instead of this License. But first, please read <https://www.gnu.org/ licenses /why-not-lgpl.html>.
+
+ </text>
+
+  
+
+LicenseID: LicenseRef-fossology-warranty-disclaimer-da410606429db1ac8820308d4a486923
```
Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:4380]: Please remove this, it i not a license

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 2]
```diff
@@ -0,0 +1,5965 @@
...
+13. Use with the GNU Affero General Public License.
+Notwithstanding any other provision of this License, you have permission to link or combine any covered work with a work licensed under version 3 of the GNU Affero General Public License into a single combined work, and to convey the resulting work. The terms of this License will continue to apply to the part which is the covered work, but the special requirements of the GNU Affero General Public License, section 13, concerning interaction through a network will apply to the combination as such.
+14. Revised Versions of this License.
+The Free Software Foundation may publish revised and/or new versions of the GNU General Public License from time to time. Such new versions will be similar in spirit to the present version, but may differ in detail to address new problems or concerns.
+Each version is given a distinguishing version number. If the Program
```
Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:4618]: Please rmove it, this is not a license

Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:4762]: Please add here:
All or portions of this licensed product
(such portions are the "Software") have been obtained under license from the
Section of Biomedical Image Analysis, Department of Radiology at the University
of Pennsylvania and are subject to the following terms and conditions:

Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:4895]: Please remove the Copyright string here

Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:5894]: Please remove the copyright string here

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.json
Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.json:1]: Here my comments on the SPDX TV file apply accordingly

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.rdf.xml
Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.rdf.xml:1]: Here my comments on the SPDX TV file apply accordingly

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.yaml
Comment (OliverFendt) [inline analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1.spdx.yaml:1]: Here my comments on the SPDX TV file apply accordingly

**Review by OliverFendt — CHANGES_REQUESTED** [review]
@ParianGol  great contribution, thank you very much. Please work on my comments

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx (hunk 2 above)
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Done, thank you!

Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Done, thank you!

Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Done, Thank you!

Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: I have removed -fossology prefix of the warranty-disclaimer.

We have to contact our lawyer for all the curated packets and we ask her our legal questions. She has insisted that we must include **warranty-disclaimer** licenses. So for this reason, I have included here.

I hope that it is ok that we keep this licence!

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 3]
```diff
@@ -0,0 +1,2270 @@
...
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+--------------------------------------------------------------------------------
+
+SBIA-License
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt:35]: Done, thank you!

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 4]
```diff
@@ -0,0 +1,2270 @@
...
+
+As a special exception to the GNU General Public License, if you distribute this
+file as part of a program that contains a configuration script generated by
+Autoconf, you may include it under the same distribution terms that you use for
+the rest of that program. This Exception is an additional permission under
+section 7 of the GNU General Public License, version 3 ("GPLv3").
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-GPL-2.0-or-later WITH Autoconf-exception-generic
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Done, thank you!

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 5]
```diff
@@ -0,0 +1,2270 @@
...
+License text of the Libtool-exception as an exception to the GPL-2.0-or-later:
+
+As a special exception to the GNU General Public License, if you distribute this
+file as part of a program or library that is built using GNU Libtool, you may
+include this file under the same distribution terms that you use for the rest of
+that program.
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-FSFULLRWD
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Done, thank you!

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 6]
```diff
@@ -0,0 +1,2270 @@
...
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This config.lt script is free software; the Free Software Foundation
+gives unlimited permision to copy, distribute and modify it.
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-Public-domain
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Done, thanks

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 7]
```diff
@@ -0,0 +1,5965 @@
...
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/install-sh
+SPDXID: SPDXRef-item495283
+FileChecksum: SHA1: 297b46f08fba07c0484ef540f9f3047d7b326cc8
+FileChecksum: SHA256: c2fd71953b47c19c62db69cbf739504aca8b47a40dbdc8c83bbc4cc7664a1779
+FileChecksum: MD5: d651f428c7739cf424b31c05dfdbed04
+LicenseConcluded: LicenseRef-fossology-Public-domain-675db48cd8d5ec259dfd9dfd1b3f1dba AND LicenseRef-fossology-X11-7bba69d17c0458741385965255d53cae
+LicenseComments: <text> The text in this file is: "FSF changes to this file are in the public domain."
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Done, Thank you!

**Review by ParianGol — COMMENTED** [review]
Thank you so much for your time and thorough explanations!

I have updated all your feedbacks except the one the ones related to warranty-disclaimer.
The issue is that our lawyer insist on handing over warranty-disclaimer licenses.

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx (hunk 1 above)
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 8]
```diff
@@ -0,0 +1,5965 @@
...
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/config.guess
+SPDXID: SPDXRef-item495278
+FileChecksum: SHA1: eebd67ddeb9453e0513a55e78888622a0ff4a175
+FileChecksum: SHA256: c081ced2d645e3b107fbf864529cc0e5954399a09b87a4f1d300470854b6dea4
+FileChecksum: MD5: 56c8530a22f26cdae7c9b5e946c06429
+LicenseConcluded: LicenseRef-fossology-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-fossology-warranty-disclaimer-90f0cbc392148555ace0fe8246603e41
+LicenseComments: <text> The information in this file is:
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:167]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 9]
```diff
@@ -0,0 +1,5965 @@
...
+Copyright 1992-2020 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/config.guess
+SPDXID: SPDXRef-item495278
+FileChecksum: SHA1: eebd67ddeb9453e0513a55e78888622a0ff4a175
+FileChecksum: SHA256: c081ced2d645e3b107fbf864529cc0e5954399a09b87a4f1d300470854b6dea4
+FileChecksum: MD5: 56c8530a22f26cdae7c9b5e946c06429
+LicenseConcluded: LicenseRef-fossology-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-fossology-warranty-disclaimer-90f0cbc392148555ace0fe8246603e41
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 10]
```diff
@@ -0,0 +1,5965 @@
...
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/config.sub
+SPDXID: SPDXRef-item495686
+FileChecksum: SHA1: 159bc91d0244e6da18c4aa4ba70065403831855b
+FileChecksum: SHA256: f08fe8f207c0fa6d722312774c28365024682333f5547c8192d0547957b000af
+FileChecksum: MD5: 190c5eaf225abd14513d768efc5073eb
+LicenseConcluded: LicenseRef-fossology-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-fossology-warranty-disclaimer-90f0cbc392148555ace0fe8246603e41
+LicenseComments: <text> The information in this file is:
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:144]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 11]
```diff
@@ -0,0 +1,5965 @@
...
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/config.sub
+SPDXID: SPDXRef-item495686
+FileChecksum: SHA1: 159bc91d0244e6da18c4aa4ba70065403831855b
+FileChecksum: SHA256: f08fe8f207c0fa6d722312774c28365024682333f5547c8192d0547957b000af
+FileChecksum: MD5: 190c5eaf225abd14513d768efc5073eb
+LicenseConcluded: LicenseRef-fossology-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-fossology-warranty-disclaimer-90f0cbc392148555ace0fe8246603e41
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 12]
```diff
@@ -0,0 +1,5965 @@
...
+
+This file is dual licensed under the terms of the MIT license
+<https://opensource.org/license/MIT>, and GPL version 3 or later
+<http://www.gnu.org/licenses/gpl-2.0.html>. You must apply one of
+these licenses when using or redistributing this software or any of
+the files within it. See the URLs above, or the file `LICENSE`
+included in the Bootstrap distribution for the full license texts.
+
+Thus we concluded additionally GPL-3-0-or-later OR MIT
+The information in this file is:
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx:76]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx [hunk 13]
```diff
@@ -0,0 +1,5965 @@
...
+##--------------------------
+
+##File
+
+FileName: jansson-2.13.1.tar.gz/jansson-2.13.1.tar/jansson-2.13.1/ltmain.sh
+SPDXID: SPDXRef-item495281
+FileChecksum: SHA1: e56cee530fdb56ca279a3f278a2bba2abaf5429d
+FileChecksum: SHA256: 48dbde49b6178a5eae0e480513ba8b36429f3503b4996e3f1be3e03444b7c575
+FileChecksum: MD5: 6ffef6c99f9e1eaa61abada7787cc095
+LicenseConcluded: LicenseRef-fossology-GPL-2.0-or-later-WITH-Libtool-exception AND LicenseRef-fossology-warranty-disclaimer-da410606429db1ac8820308d4a486923 AND (LicenseRef-GPL-3.0-or-later OR LicenseRef-MIT)
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-SPDX2TV.spdx]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 14]
```diff
@@ -0,0 +1,2270 @@
...
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-warranty-disclaimer
+
+This is free software; see the source for copying conditions. There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE."
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-warranty-disclaimer
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

File: analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt [hunk 15]
```diff
@@ -0,0 +1,2270 @@
...
+---------------------
+
+This Agreement shall be governed by and construed in accordance with the laws
+of The Commonwealth of Pennsylvania without regard to principles of conflicts
+of law. This Agreement shall supercede and replace any license terms that you
+may have agreed to previously with respect to Software from SBIA.
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-warranty-disclaimer
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Thank you for your review! We have forwarded exactly this example to our lawyer and she mentioned that we must hand over warranty-disclaimer license. How do you recommend to proceed in this case?

[the remark this answers is not in the document]
Comment (OliverFendt) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: ok, I see, in this case let's move on an keep this in the documents.
Did you lawyer also provide an explanation why she is of the opinion to keep it?

[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/jansson/version-v2.13.1/jansson-v2.13.1-OSS-disclosure.txt]: Thank you!
She said exactly the sentences below:
"Der Entwickler könnte sich etwas dabei gedacht haben. Warranty-Disclaimer sollen immer beibehalten werden."

I can still ask her for more explanations, when she responds, I will write to you!

**Review by OliverFendt — APPROVED** [review]
@ParianGol wunderful. Thank you so much for you efforts.
In case you have the opportunity please say thank you to your lawyer for her efforts too.

**Pull request merged**